tests/e2e/vsan_test.go

```diff
@@ -101,3 +103,41 @@ func (s *VsanTestSuite) TestVSANPolicy(c *C) {
 
 	misc.LogTestStart(volCreateTest, "TestVSANPolicy")
 }
+
+// Test step:
+// 1. create a vsan policy
+// 2. run "vmdkops_admin policy ls", check the "Active" column of the output to make sure it
+// is shown as "Unused"
+// 3. create a volume on vsanDatastore with the vsan policy we created
+// 4. run "docker volume inspect" on the volume to verify the output "vsan-policy-name" field
+// 5. run "vmdkops_admin policy ls", check the "Active" column of the output to make sure it
+// is shown as "In use by 1 volumes"
+// 6. run "vmdkops_admin policy rm" to remove the policy, which should fail since the volume is still
+// use the vsan policy
+func (s *VsanTestSuite) TestDeleteVsanPolicyAlreadyInUse(c *C) {
```

Comment:
Since we are adding more VSAN related test cases, let's also rename the existing test case "TestVSANPolicy" - the name is too generic. In fact we should split it into two.

I know this request is out of the scope of this PR, but I think the change is straightforward, so I'd suggest to fix it.